amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -96,84 +93,68 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
-           </div>
-        </div>
-      );
-    }
-    if (!this.state.inEditMode || (this.state.inEditMode && this.state.isDisabled)) {
-      return (
-        <div id='editable-container' className='editable-container'>
-          <Overlay
-            placement='top'
-            show={ this.state.isDisabled }
-            target={ this.getAnchorTarget }
-          >
-            <Tooltip id='error-tooltip'>
-              <div className="error-tooltip">
-                <text>This text is out of date, please refresh the component</text>
-                <button onClick={ this.refreshText } className="btn btn-flat-icon">
-                  <img className='icon icon-refresh'/>
-                </button>
-              </div>
-            </Tooltip>
-          </Overlay>
-          <div id='editable-text' className="editable-text">
-            <ReactMarkdown source={ this.state.value }/>
-            <a className={ "edit-link" + (this.state.value ? "" : " no-value") }
-               href="JavaScript:void(0)"
-               onClick={ this.enterEditMode }
-               ref={ this.editAnchorRef }
-            >
+             </div>
+             {
+               this.state.editable &&
+               <a className={"edit-link" + (this.state.value ? "" : " no-value")}
+                  href="JavaScript:void(0)"
+                  onClick={this.enterEditMode}
+               >
+                 {
+                   this.state.value ? "edit" : "Add Description"
+                 }
+               </a>
+             }
+           </>
+        }
+        {
+          this.state.inEditMode &&
+          <>
+            <textarea
+              className='editable-textarea'
+              rows={ 2 }
+              maxLength={ this.props.maxLength }
+              ref={ this.textAreaRef }
+              defaultValue={ this.state.value }
+            />
+            <div className="editable-textarea-controls">
               {
-                this.state.value ? "edit" : "Add Description"
-              }
-            </a>
-          </div>
-        </div>
-      );
-    }
-
-    return (
-      <div id='editable-container' className='editable-container'>
-        <textarea
-          id='editable-textarea'
-          className='editable-textarea'
-          rows={ 2 }
-          maxLength={ this.props.maxLength }
-          ref={ this.textAreaRef }
-          defaultValue={ this.state.value }
-        />
-        <Overlay
-          placement='top'
-          show={ true }
-          target={ this.getTextAreaTarget }
-        >
-          <Tooltip id='save-tooltip'>
-            <button id='cancel' onClick={this.exitEditMode}>Cancel</button>
-            <button id='save' onClick={this.updateText}>Save</button>
-          </Tooltip>
-        </Overlay>
-      </div>
+                this.state.isDisabled &&
+                 <>
+                   <p>This text is out of date, please refresh the component</p>
```

Comment:
What do you think of styling this to be the `danger` color?

Reply:
How does this look?

<img width="329" alt="Screen Shot 2019-10-29 at 4 30 42 PM" src="https://user-images.githubusercontent.com/4338702/67817106-86b48b80-fa69-11e9-914a-1fe2c598f5f8.png">

Reply:
Kind of weird actually because this presents itself like a badge, but it's not a bagde. I was initially imagining red text, but I remember our danger color is actually a softer `$sunset` color and not red. I suppose let's use best judgement if there is anyway to highlight the text without co-opting a design concept that was created for another purpose?

I'm alright if we have to leave this as an open question for design sync cleanup.